app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class BookSerializer < ActiveModel::Serializer
+  attributes :id
```

Comment:
Usually a serializer will have more fields than just the ID. Are you sure we don't need to show other fields? Book title, author, etc?